pyppeteer_test.py

```diff
@@ -0,0 +1,32 @@
+import os
+import asyncio
+import time
+import sys
+from pyppeteer import launch
+
```

Comment:
Simple script that checks that the built JS output is good and has the solr search fields we expect.  Confirmed that it fails when the search fields aren't present.